packages/forms/README.md

```diff
@@ -138,6 +138,18 @@ You can use them by importing the DataPropTypes and ActionsPropTypes functions.
 import Form, { DataPropTypes, ActionsPropTypes } from '@talend/react-forms'
 ```
 
+### Validation
+
+You can use validation from outside (let say button outside the form) this way:
+
+```javascript
+import validate from '@talend/react-forms/lib/validate';
+
+function isvalide(options) {
```

Comment:
isValid

how do i use isValid with a specific form ?
Or is this not intented to allow to supersed the basic form validation mechanism ?

Reply:
your specific form is using jsonSchema and has data. But those can be anywhere (in a redux state or not).
In some way the json schema can be built on the fly by your component.
So this is why it's considered as external validation.
You may want to disable a button outside the form when the form is not valid.

So our form lib will use it's own validation mecanism internally to display error results.
Here we already have two libs. I have played with thoses two libs and have issues on validations.
So here I propose to use ajv which is far better, faster and looks good at the errors.
May be one day our form lib will use ajv ?